pagopa/io-app
# feat: [IOCOM-1406] Toast handling on new DS's Messages Home

⚠️ This PR depends on #5854 and #5867 ⚠️

## Short description
This PR refactors the toast visualisation in the new messages' home screen.

## List of changes proposed in this pull request
- A single `Toast` component is added to the new `MessagesHomeScreen`
- Such component handles all the toast that have to be shown, split by their business-case
- Toast visibility and data are retrieved using the new selectors in `allPaginated`

## How to test
Using the io-dev-api-server and making sure that both new DS and new Home are enabled, retrieve some messages. Stop and restart the server, changing the configuration for the `GET messages` endpoint status code response (with an error one). Try to refresh or load more messages and see that the proper toast is displayed).
